inferentia/scripts/gen_triton_model.py

```diff
@@ -270,20 +271,27 @@ def get_tensorflow_initialize_impl():
                 (config_output['name'], config_output['data_type'],
                  config_output['dims']))
 
-        # TODO: Validate input/output from the model
-
         os.environ["NEURON_RT_NUM_CORES"] = str(cores_per_instance)
-
+'''
+    if is_inf2:
+        init_impl += '''
+        compiled_model = os.path.join(args['model_repository'], compiled_model)
+        self.pred_list = [
+        tf.keras.models.load_model(compiled_model)
+        for _ in range(cores_per_instance)
+        ] * threads_per_core 
+'''
+    else:
+        init_impl += '''
         self.pred_list = [
-            tf.contrib.predictor.from_saved_model(compiled_model)
-            for _ in range(cores_per_instance)
+        tf.contrib.predictor.from_saved_model(compiled_model)
```

Comment:
miss indentation here. You will get `PEP 8: E122 continuation line missing indentation or outdented` when running format check on generated `model.py`

Reply:
Thanks, updated